core/src/test/resources/HistoryServerExpectations/incomplete_app_executor_list_json_apply_custom_log_urls_apply_incomplete_app_false_expectation.json

```diff
@@ -0,0 +1,121 @@
+[ {
```

Comment:
Not needed?

Reply:
Nice finding. I found 4 files which can be deleted. Will delete.